test/DepositWithAuthorizationAndOperatorApproval.t.sol

```diff
@@ -98,7 +98,7 @@ contract DepositWithAuthorization is Test, BaseTestHelper {
         );
 
         // Corrupt r
-        r = bytes32(uint256(r) ^ 1);
+        r = bytes32(uint256(r) ^ 3);
```

Comment:
https://github.com/FilOzone/filecoin-pay/pull/229/files#r2356494903

Changing the private key made this signature valid. This change made it invalid again.